### DISCUSSION TITLE
[M-Z] (9 Settembre 2019) Esercizio 5

### DISCUSSION THREAD

**Discussion by rimaout** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#481] [Esami & Esoneri M-Z]
### Traccia

<img width="933" alt="image" src="https://github.com/user-attachments/assets/abb012cc-5249-4ad9-838e-93c001ae64bc">

**Comment by matypist** [comment]
Segue una proposta di soluzione di Andrea Princic (@Princic-1837592), tratta dal suo file di soluzioni [20190909.pdf](https://raw.githubusercontent.com/Princic-1837592/metodi-matematici/refs/heads/main/soluzioni/20190909.pdf) nel suo [repository GitHub](https://github.com/Princic-1837592/metodi-matematici) contenente le sue soluzioni agli esami del prof. Piperno:

![image](https://github.com/user-attachments/assets/11cdfe05-2423-4e4d-8cd6-5bd7d687655e)
